plugins/woocommerce/src/Blocks/BlockTypes/ProductGallery.php

```diff
@@ -125,16 +125,20 @@ protected function render( $attributes, $content, $block ) {
 				'data-wp-context',
 				wp_json_encode(
 					array(
-						'imageData'         => $image_src_data,
-						'isDialogOpen'      => false,
-						'disableLeft'       => true,
-						'disableRight'      => false,
-						'isDragging'        => false,
-						'touchStartX'       => 0,
-						'touchCurrentX'     => 0,
-						'productId'         => $product_id,
-						'selectedImageId'   => $initial_image_id,
-						'userHasInteracted' => false,
+						'imageData'                => $image_src_data,
+						'isDialogOpen'             => false,
+						'disableLeft'              => true,
+						'disableRight'             => false,
+						'isDragging'               => false,
+						'touchStartX'              => 0,
+						'touchCurrentX'            => 0,
+						'productId'                => $product_id,
+						'selectedImageId'          => $initial_image_id,
+						'userHasInteracted'        => false,
+						'thumbnailsOverflowTop'    => false,
+						'thumbnailsOverflowBottom' => false,
+						'thumbnailsOverflowLeft'   => false,
+						'thumbnailsOverflowRight'  => false,
```

Comment:
Nitpick: Thinking of it again, I wonder if pushing those attributes to an object 
```
thumbnailsOverflow: {
 top:false,
 bottom: false.
 left: false,
 right: false
}
```

would be more readable and easier to pass in context.

Reply:
Actually it does make sense! https://github.com/woocommerce/woocommerce/pull/56069/commits/a5b1bb63ba98ce15644af8176595aaf16e185756